doc/daikon.texinfo

```diff
@@ -260,7 +260,7 @@ There are two additional requirements.  You must have a Java 7 (or later) JVM
 The @file{tools.jar} file that comes with your JVM must also be on your
 classpath.
 
-@xref{Installing Daikon}, for situations where you should
+See bullet point two above (@ref{Installing Daikon}), for situations where you should
 follow the complete installation instructions of
 @ref{Complete installation}.  (Also, if you do not know how to add a jar file to your
```

Comment:
Isn't this documentation now obsolete, along with anything else that mentions the complete installation as a contrast to other approaches?